libs/ui/src/components/NeoModal/NeoModal.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <teleport to="body" :disabled="!appendToBody">
+  <teleport v-if="isModalActive" to="body" :disabled="!appendToBody">
```

Comment:
can we do a local solution for each modal

Reply:
Why apply it for each modal? It's kind of a bug from `OModal` that the DOM of modal will not be removed after closing the modal.

Reply:
pls [check](https://github.com/kodadot/nft-gallery/pull/10306#issuecomment-2120333504) 

@daiagi what do you think.